src/bounds.jl

```diff
@@ -146,3 +146,51 @@ a definite sign.
 
 """
 bound_taylor1(fT::TaylorModel1, I=domain(fT)::Interval) = bound_taylor1(polynomial(fT), I)
+
+"""
+    linear_dominated_bounder(fT::TaylorModel1, ϵ=1e-3::Float, max_iter=5::Int)
+
+Compute a tighter polynomial bound for the Taylor model `fT` by the linear
+dominated bounder algorithm. The linear dominated algorithm is applied until
+the bound of `fT` gets tighter than `ϵ` or the number of steps reachs `max_iter`.
+"""
+function linear_dominated_bounder(fT::TaylorModel1; ϵ=1e-3, max_iter=5)
```

Comment:
It seems to me that the function *assumes* that `fT` is a `TaylorModel1{Float64,Float64}`, i.e., where the coefficients of the polynomial are `Float64`.

Do you know if it also would work for `TaylorModel1{Interval{Float64},Float64}`? If it doesn't, maybe it is a good idea to have a more specific signature.

Reply:
The current implementation doesn't work for a `TaylorModel1` with interval coefficients. I'm working on it.